src/Symfony/Component/WebLink/Link.php

```diff
@@ -134,7 +134,7 @@ public function withoutRel($rel)
      *
      * @return static
      */
-    public function withAttribute($attribute, $value)
+    public function withAttribute(string $attribute, string|\Stringable|int|float|bool|array $value)
     {
         $that = clone $this;
         $that->attributes[$attribute] = $value;
```

Comment:
Lets update the type for the `$attribute` property too. It currently says it should be an array of strings.